src/content/docs/licenses/end-of-life/notification-changes-new-relic-saas-features-distributed-software.mdx

```diff
@@ -38,74 +38,104 @@ EOL changes are classified into the following categories:
 <table>
   <thead>
     <tr>
-      <th style={{ width: "150px" }}>
-        Transition impact
+      <th style={{ width: "80px" }}>
+        Tier
       </th>
 
-      <th>
-        Description
+      <th style={{ width: "240px" }}>
+        Impact to customer
       </th>
 
-      <th style={{ width: "200px" }}>
-        Notification lead time
+      <th style={{ width: "240px" }}>
+        Risk of customer not acting
+      </th>
+      <th style={{ width: "80px" }}>
+        Impact
       </th>
+      <th style={{ width: "115px" }}>
+        Notification lead time
+      </th>      
     </tr>
   </thead>
 
   <tbody>
     <tr>
       <td>
-        Low
+        Tier 3
       </td>
-
       <td>
-        Limited changes to customer processes or procedures are anticipated.
+        None or Cosmetic only (cannot affect customer workflow or platform usage)
+      </td>
+      <td>
+        None
       </td>
 
       <td>
-        None to 90 days
+        Low (Bulk)
+      </td>
+      <td>
+        None
       </td>
     </tr>
 
     <tr>
       <td>
-        Moderate
+        Tier 2
+      </td>
+      <td>
+        Customer workflow may be impacted
       </td>
-
       <td>
-        Changes to functionality are limited in scope. Actions such as upgrading retired agents to supported versions may be required.
+        None
       </td>
 
+      <td>
+        Moderate (Bulk)
+      </td>
       <td>
         Minimum of 90 days
       </td>
     </tr>
 
     <tr>
       <td>
-        High
+        Tier 1
       </td>
 
       <td>
-        Changes are highly impactful to customers and will require significant investment from customers to adapt.
+        * Customer configuration/coding changes required
+        * Capability being replaced (including APIs, agents, and integrations)
       </td>
 
+      <td>
+        * Data integrity will be affected including partial or full data loss
+        * Alerts impacted including false positive and false negatives
+      </td>
+      <td>
+        High
+      </td>
       <td>
         Minimum of 180 days
       </td>
     </tr>
 
     <tr>
       <td>
-        Critical
+        Tier 1A
       </td>
 
       <td>
-        A third party dependency, security issue, or other critical risk causes an urgent need to change or remove functionality on an accelerated timeline.
+        There is third party dependency, security issue, reliability concern, or other critical risk that causes an urgent need to change or remove functionality on an accelerated timeline.      
       </td>
 
       <td>
-        Notified as soon as reasonably practical
+        See entry to left
```

Comment:
This seems odd, not sure what it means. I read the entry to the left, but I'm still not sure what the customer risk is.